test/integration/openid4ci_test.go

```diff
@@ -104,7 +107,9 @@ func TestOpenID4CIFullFlow(t *testing.T) {
 		c, err := did.NewCreatorWithKeyWriter(kms)
 		require.NoError(t, err)
 
-		didDoc, err := c.Create(tc.walletDIDMethod, &api.CreateDIDOpts{})
+		didDoc, err := c.Create(tc.walletDIDMethod, &api.CreateDIDOpts{
+			KeyType: tc.walletKeyType,
```

Comment:
What's the default key type ? Also, can we add the list of supported key types to the doc https://github.com/trustbloc/wallet-sdk/blob/main/cmd/wallet-sdk-gomobile/docs/usage.md#examples-2 ?

Reply:
Default key type is Ed25519

Reply:
Pushed with updated doc!